pootle/checks.py

```diff
@@ -547,3 +547,26 @@ def check_mysql_timezones(app_configs=None, **kwargs):
             if converted_now is None:
                 errors.append(missing_mysql_timezone_tables)
     return errors
+
+
+@checks.register()
+def check_mysql_collation(app_configs=None, **kwargs):
+    from django.db import connection
+
+    non_utf8_collation_and_character_set = checks.Critical(
+        _("MySQL charset and collation must support UTF8."),
+        hint=("Change charset and collation settings for you MySQL database"),
+        id="pootle.C023",
+    )
+    errors = []
+    with connection.cursor() as cursor:
+        import pdb; pdb.set_trace()
+        if hasattr(cursor.db, "mysql_version"):
+            sql = ("SELECT default_character_set_name, default_collation_name "
+                   "FROM information_schema.SCHEMATA WHERE schema_name = '%s'" %
+                   cursor.db.settings_dict['NAME'])
+            cursor.execute(sql)
+            character_set, collation = cursor.fetchone()
+            if 'utf8' not in character_set or 'utf8' not in collation:
+                errors.append(non_utf8_collation_and_character_set)
```

Comment:
also - im thinking that some dbs may have custom encoding for relevant cols - eg utfmb4 iirc